### ISSUE TITLE
Nano: IPEX 1.10 is needed to run LSTM

### ISSUE THREAD

**Issue by zhentaocc** [intel-analytics/BigDL#3617]
LSTM suffers from this issue https://github.com/intel/intel-extension-for-pytorch/issues/187  
Old version of ipex cannot correctly handle the weight reshaping from xpu to cpu.  
I got reply that this is resolved in 1.10. So I think we need to update our environment for nano.   
But I don't see IPEX as pytorch dependency in setup. 
Possibly need to upgrade torch to 1.10 too. IPEX team doesn't tell if torch is compatible with torch <1.10.
Seems right now we can only install from source.  We might need to do this after 1.10 officially released. @yangw1234

**Comment by zhentaocc** [comment]
FYI @TheaperDeng @shane-huang

**Comment by TheaperDeng** [comment]
thx, that's good news. 1.10 will be released soon. The concern is that 1.10 will move cpu computation from xpu back to cpu. This may cause some coding adaptation.

**Comment by TheaperDeng** [comment]
I have completed some experiment on ipex1.10(directly install from whl relased [here](https://github.com/intel/intel-extension-for-pytorch/releases))

under same data and hw env we benchmarked
For LSTM, 364s/epoch (ipex) vs 634s/epoch (no ipex), but the loss does not convergence as expected.
For TCN I changed weightnorm to batchnorm, 421s/epoch (ipex) vs 526s/epoch (no ipex).

I think the result is promising but we should have another talk with @yangw1234 later since there is a refactor on API and we need to redesign(if we need to adopt `pytorch_lightning>=1.6`)/reimplement our `IPEXAccelerator`.

Here I suggest to put a hold on ipex1.10 and this comment is just a record for future.
